nslsii/sync_experiment/sync_experiment.py

```diff
@@ -111,9 +111,15 @@ class AuthorizationError(Exception): ...
 
 def sync_experiment(proposal_number, beamline, verbose=False, prefix=""):
 
-    redis_client = redis.Redis(host=f"info.{beamline.lower()}.nsls2.bnl.gov")
+    normalized_beamlines = {
+        "sst1": "sst",
+        "sst2": "sst",
+    }
+    redis_beamline = normalized_beamlines.get(beamline.lower(), beamline)
+    redis_client = redis.Redis(host=f"info.{redis_beamline.lower()}.nsls2.bnl.gov")
 
-    md = RedisJSONDict(redis_client=redis_client, prefix=prefix)
+    redis_prefix = f"{prefix}-" if prefix else ""
```

Comment:
Is there a reason we add "-" to the prefix here? If this is a redis's syntectic requirement, would it be better to handle this by `RedisJSONDict` itself? Right now, to run `sync-experiment`, one uses `-e swaxs` (without the '-'), but in the profile collection, we have to set `prefix='swaxs-'` when we initialize the redis client, which is confusing and error-prone.

Reply:
I don't have a strong opinion, particularly if implemented this way (i.e. it is optional to use) I don't think it can hurt.

Reply:
I think specifying a delimiter with a default value set to "-" makes a lot of sense!